### DISCUSSION TITLE
2022年11月起，华工研究生毕业论文格式更新

### DISCUSSION THREAD

**Discussion by xinwuchn** [mengchaoheng/SCUT_thesis#39] [General]
如题，华工毕业论文模板好像更新了格式，请问大佬该模板是否满足新的格式要求～

**Comment by note286** [comment]
不要用好像这种词语描述，如果有更新，给出规范文件或者链接等信息。

**Comment by xinwuchn** [comment]
嗯呢，这里是新的撰写规范～
[华工论文写作规范2022.11.pdf](https://github.com/mengchaoheng/SCUT_thesis/files/11106855/2022.11.pdf)

**Reply by mengchaoheng** [reply]
我的文档里给出了过去的两份撰写规范，你花三分钟对一下就知道到底有没有更新，还是只加上了“更新”两个字并改了日期而已。然而你选择来问我…，都是读研究生的人了。同学要相信自己！对完了有发现跟我们分享一下，谢谢！

**Reply by xinwuchn** [reply]
非常抱歉哈，我没注意到您的文档里给出的撰写规范。

**Reply by mengchaoheng** [reply]
哈哈哈，我其实很期待他真的改了，我师弟对了，还没回我改没改。已经好多年只改日期了

**Comment by nantayo** [comment]
@mengchaoheng hello, 我是数学学院的, 注意到我们下发的封面文件使用了新的校徽, 不知道其他学院有无更改?
![new](https://user-images.githubusercontent.com/77558823/230767714-f1a5dd33-5d90-4941-822d-944ea8d25467.png)
↑这是新的
↓这是旧的
![old](https://user-images.githubusercontent.com/77558823/230767716-461764fd-3143-44b3-99c3-516e18405dff.png)

**Reply by sikouhjw** [reply]
除了大小外，哪里改了？

**Reply by note286** [reply]
时间变了，一个是1918，一个是1934。

**Reply by mengchaoheng** [reply]
我有个疑问，你们自行把封面换了不就好了吗？不涉及格式问题吧？前面有个帖子说到的段前短后的距离问题，倒是还没搞清楚。

**Reply by nantayo** [reply]
> 我有个疑问，你们自行把封面换了不就好了吗？不涉及格式问题吧？前面有个帖子说到的段前短后的距离问题，倒是还没搞清楚。

的确,一个很小的改动,不过还是说一声,不然的话肯定有人下下来模板不看的,然后不改

**Reply by mengchaoheng** [reply]
嗯嗯是的，改年份这个事，有意思哈哈哈

**Reply by nantayo** [reply]
> 嗯嗯是的，改年份这个事，有意思哈哈哈

无论如何致敬你们的贡献, 模板已经很好用了.😇🙏
现在来看, 唯一的问题就是, 导师用CTeX的还是太多了......
